### PR TITLE
Don't enter multi-domain unless it's configured

### LINKED ISSUE #868: Don't do multi-domain unless there is a domain configured
Also add the possibility to disable it even if there are domains configured so that when domains config in PacketFence will be used for something else than FreeRADIUS, it still won't do it

Idea is to make it conditional in the template generation on service start.

Also need to add message to restart FreeRADIUS after a change

### PR DESCRIPTION
**Pull request by julsemaan**
# Description

Doesn't activate the multi domain FreeRADIUS module unless there are domains configured in PacketFence

Also adds the possibility to disable it even if there are domains configured so that when domains config in PacketFence will be used for something else than FreeRADIUS, it still won't do it

Remove the call to packetfence-multi-domain in the outer tunnel
# Impacts

FreeRADIUS domain authentication
# Issue

fixes #868 
# Delete branch after merge

YES
# NEWS file entries
## Enhancements
- Will not activate the packetfence multi-domain FreeRADIUS module unless there are domains configured in PacketFence
- Added option to disable the packetfence multi-domain FreeRADIUS module from the admin interface

### CODE REVIEW

File: conf/documentation.conf [hunk 1]
```diff
@@ -1103,6 +1103,14 @@ The cost factor to apply to the password hashing if applicable.
 Currently only applies to bcrypt.
 EOT
 
+[advanced.disable_pf_domain_auth] 
+type=toggle
+options=enabled|disabled
+description=<<EOT
+Disables authentification on the domains defined in PacketFence.
+This will make it so the authentication only happens on the domain binded to the OS.
```
Comment (louismunro) [inline conf/documentation.conf]: Not sure this message is clear enough for a user that is not familiar with the FreeRADIUS/PacketFence/Winbind internals.
Maybe we should say something like 'disable Active-Directory domain configuration from within PacketFence'.

File: conf/pf.conf.defaults [hunk 2]
```diff
@@ -851,6 +851,12 @@ hash_passwords=bcrypt
 # The cost factor to apply to the password hashing if applicable.
 # Currently only applies to bcrypt.
 hashing_cost=8
+#
+# advanced.disable_pf_domain_auth
+#
+# Disables authentification on the domains defined in PacketFence
+# This will make it so the authentication only happens on the domain binded to the OS.
```
Comment (louismunro) [inline conf/pf.conf.defaults]: Same here.

**Comment by louismunro** [comment]
LGTM apart from minor comments and documentation nitpicking.

**Comment by louismunro** [comment]
Please fix the (minor) comment and documentation issues.
I will merge as soon as that is done.

**Comment by julsemaan** [comment]
Sorry forgot about this one, I'll fix in the next minutes

**Comment by julsemaan** [comment]
Fixaide

**Pull request merged**